### PR TITLE
[NUI] Add RotationCompletedAcknowledgement APIs

### PR DESCRIPTION
**Pull request by sparrow74**
Add RotationCompletedAcknowledgement APIs in Window.
The APIs are SetNeedsRotationCompletedAcknowledgement and SendRotationCompletedAcknowledgement

### CODE REVIEW

**Review by dongsug-song — APPROVED** [review]
approved

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs [hunk 1]
```diff
@@ -18,12 +18,20 @@ public class WindowTest : IExample
 
         int addingInput;
         Timer tm;
+        bool manualRotation;
+        int rotation_count;
```
Comment (dkdk-ryu) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs:22]: I think that rotationCount is better.

Comment (sparrow74) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs:22]: I see, I will replace with rotationCount

**Review by taehyub — APPROVED** [review]
LGTM :)

**Review by dkdk-ryu — APPROVED** [review]

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**